### PR TITLE
LPD-24688 Fix space between the input and the message

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPD-24688)

## Steps 

- Activate the release FF: Translation Status for Web Content Fields (LPS-114700)
- Go to the Web content section
- Create a new web content
- Select a different Language

NOTE: I have added a negaive margin in order to not change the margin for all "<liferay-friendly-url:input>" taglib use cases

### CODE REVIEW

**Review by ambrinchaudhary — CHANGES_REQUESTED** [review]
Hi @DiegoHu97, we are not fans of negative margins :D 

The <p> tag is the one that is adding an extra margin, why don't you just replace is by a <span>? 
I've tried and this is how it looks like, for me this approach is better

**Review by ambrinchaudhary — CHANGES_REQUESTED** [review]
Hey @DiegoHu97, we are not big fans of negative margins ;-) 

I see there is a `helpMessage` in the <liferay-friendly-url:input> taglib, could we use it for that message? 

 If not, is it possible to remove the `margin-bottom` for the parent field? Could be by css
```
.friendly-url{
	margin-top: 0;
}
```

or adding a `mb-0` class to the taglib?

**Review by DiegoHu97 — COMMENTED** [review]
I added the margin to the .friendly-url class

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -160,6 +160,10 @@ $productMenuWidth: 320px;
 			@include media-breakpoint-between(sm, md) {
 				margin-top: $toolbarHeight;
 			}
+
+			.friendly-url {
+				margin-bottom: 4px;
```
[the remark this answers is not in the document]
Comment (DiegoHu97) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/css/main.scss]: Done! Thanks! @ambrinchaudhary

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by DiegoHu97** [comment]
ci:report:2566184924

**Comment by DiegoHu97** [comment]
ci:report:2566184967

**Comment by DiegoHu97** [comment]
ci:forward:force

**Pull request closed without merging**